src/Card/CardImageCap.jsx

```diff
@@ -45,26 +46,37 @@ const CardImageCap = React.forwardRef(({
     );
   }
 
-  const handleSrcFallback = (event, altSrc) => {
+  const handleSrcFallback = (event, altSrc, imageKey) => {
     const { currentTarget } = event;
-    if (currentTarget.src !== altSrc) {
-      currentTarget.src = altSrc;
+
+    if (!altSrc || currentTarget.src.endsWith(altSrc)) {
+      if (imageKey === 'imageCap') {
+        currentTarget.src = cardSrcFallbackImg;
```

Comment:
Good idea here to fall back to the default `cardSrcFallbackImg` in this case! Should we ensure this line gets covered in the tests?

Reply:
Done ✅